libs/components/indicators/src/lib/modules/tokens/tokens.component.scss

```diff
@@ -19,10 +19,14 @@ $sky-tokens-gutter-size: 2px;
     flex-grow: 1;
     flex-basis: 0px;
 
-    // Add a zero-width space to allow form controls to align
-    // properly to text baseline.
-    &:before {
-      content: "#{'\200B'}";
+    > div {
+      display: contents;
```

Comment:
I'm seeing some warnings around accessibility with the `display: contents` feature. Should we be concerned about that? https://caniuse.com/?search=display%3A%20contents

Reply:
The Safari issues are concerning, the fix landed in Safari Technology Preview 174, but there is potential to break accessibility for prior versions. I don't have a good assessment of how it compares the current issue and would have to dig into that.

Chrome and Firefox implementations seem to be fine.

Reply:
I've confirmed that the current implementation makes that area inaccessible in Safari.